pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Link to JIRA ticket
+---------------------------------------------------
+
+Add a link to the JIRA ticket that the changes in this PR describes.
+
+## Describe this PR
+---------------------------------------------------
+
+### *What is the problem we're trying to solve*
+
+Add a summary of the problem and any additional context that helps explain what the issue we're trying to solve is.
+
+### *What changes have we introduced*
+
+Add descriptions of what changes were made to address the problem and provide a solution.
+
+Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.
+
+#### _Checklist_
```

Comment:
Can we have a point about having an integration test if necessary, please

Reply:
Yep, will add that in 👍